### PR TITLE
3.40.5

### PR DESCRIPTION
**Pull request by ericgaspar**
- *Upgrade to version 3.40.2*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/ghost_ynh%20PR30%20(ericgaspar)/)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: check_process [hunk 1]
```diff
@@ -17,24 +17,15 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		# 2.1.1
-		upgrade=1	from_commit=d82ce7f51218a4ec32acedbe643b1168721786d0
 		# 3.19.2
 		upgrade=1	from_commit=cf8c367214ef277f40a43489365199725540e021
```
Comment (yalh76) [inline check_process:21]: Would be great to add a test from current 3.38.2

File: scripts/restore [hunk 2]
```diff
@@ -107,14 +107,14 @@ ynh_mysql_connect_as --user=$db_user --password=$db_pwd --database=$db_name < ./
...
-systemctl enable $app.service
+systemctl enable $app.service --quiet
 
 #=================================================
 # INTEGRATE SERVICE IN YUNOHOST
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
-yunohost service add $app --description "$app daemon for Ghost" --log_type "systemd"
+yunohost service add $app --description="$app daemon for Ghost" --log="/var/log/$app/$app.log"
```
Comment (yalh76) [inline scripts/restore:117]:
```suggestion
yunohost service add $app --description "$app daemon for Ghost" --log "/var/log/$app/$app.log"
```
Not sure the `=` will work

File: scripts/upgrade [hunk 3]
```diff
@@ -211,7 +194,7 @@ chown -R $app: $final_path
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
-yunohost service add $app --description "$app daemon for Ghost" --log_type "systemd"
+yunohost service add $app --description="$app daemon for Ghost" --log="/var/log/$app/$app.log"
```
Comment (yalh76) [inline scripts/upgrade:197]:
```suggestion
yunohost service add $app --description "$app daemon for Ghost" --log "/var/log/$app/$app.log"
```

Not sure there either ...

**Review by yalh76 — APPROVED** [review]

File: scripts/restore (hunk 2 above)
Comment (ericgaspar) [reply scripts/restore:117]: `=` work! and they arealso  part of *example_ynh*

**Pull request merged**